### DISCUSSION TITLE
TeamTalk 5.14 Ubuntu18 Is Not Shown On the Website or In the TeamTalk Directory

### DISCUSSION THREAD

**Discussion by ChrisDuffley** [BearWare/TeamTalk5#1991] [Q&A]
### Description

I consider this a bug because Ubuntu 20.04 LTS is still supported and is based off of the dependencies for Ubuntu 18 and is also still used in Debian, as far as I understand it.

### Application

- [ ] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [X] TeamTalkServer

### Platform

- [ ] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [X] Linux

### Steps to reproduce problem

Look on either the [TeamTalk directory](https://bearware.dk/teamtalk/v5.14) and/or also the main download page, and you will see that it's not there. This means that only people using Ubuntu 22.04 or later can use the newer version.

**Comment by jeremyp3** [comment]
Hello,

since the implementation of ssl certificates, #1780 it is no longer possible to build Teamtalk on qt versions lower than 5.15.

that's why ubuntu18 is no longer available for teamtalk.

**Comment by bear101** [comment]
Ubuntu 18 is only supported by TeamTalk Pro Server. So if you want to use Ubuntu 18 (or 20) you can either:
- Purchase a TeamTalk Pro License
- Upgrade your server to Ubuntu 22
- Run TeamTalk server in Docker using Ubuntu 22 image
